scripts/convert-to-json.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+const path = require('path');
+
+(async () => {
+  const data = fs.readFileSync(path.resolve(__dirname, './tlds'), 'utf8')
```

Comment:
Yeah, feels like this should download the list vs. keeping two copies of TLDs around in the repo.